fxprof-processed-profile/src/marker_table.rs

```diff
@@ -199,6 +218,12 @@ impl Serialize for SerializableMarkerDataElement<'_> {
                     (value, number_fields) = number_fields.split_first().unwrap();
                     map.serialize_entry(&field.key, value)?;
                 }
+                MarkerFieldFormatKind::Flow => {
+                    let value;
+                    (value, flow_fields) = flow_fields.split_first().unwrap();
+                    // Flow IDs are serialized as hex strings without 0x prefix
+                    map.serialize_entry(&field.key, &format!("{:x}", value))?;
```

Comment:
This needs to actually use the string table and output the string index.

Reply:
Ah, it probably needs to happen in `add_marker`, where we call `flow_field_value`.

Reply:
So add_marker would need a mut reference to the Profile then?